GetandTake/Program.cs

```diff
@@ -1,10 +1,24 @@
+using AutoMapper;
+using GetandTake.DataAccess.Repositories;
+using GetandTake.DataAccessLayer.EF;
+using GetandTake.Services.Abstracts;
+using GetandTake.Services.AutoMapper;
+using GetandTake.Services.Concretes;
+using Microsoft.EntityFrameworkCore;
+using System;
+
 var builder = WebApplication.CreateBuilder(args);
 
+builder.Services.AddDbContext<NorthwindDbContext>(
+    options =>  options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection")));
 // Add services to the container.
 builder.Services.AddRazorPages();
```

Comment:
Let's split your confiration by separate files, it will simplify future configuration: please, add folder `Configuration` and declare there 2 files called `ServicesConfiguration` and `MiddlewareConfiguration`. The second one will be used later, we can leave it empty (or you can add it in future by yourself, up to you). And add [such](https://github.com/dotnet-fizzyy/Diploma/blob/V2/WebAPI/WebAPI/Startup/Startup.cs#L28) method to your `ServicesConfiguration` class file :)